cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/ExecutorConfig.java

```diff
@@ -62,6 +65,9 @@ public static ExecutorConfig create(
             @JsonProperty("cache_fetched_uris") boolean cacheFetchedUris)
             throws URISyntaxException, UnsupportedEncodingException {
 
+        if (libmesosLocation == null || StringUtils.isEmpty(libmesosLocation)) {
```

Comment:
`StringUtils.isEmpty()` already includes a `null` check, the `null` check here shouldn't be needed: https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#isEmpty(java.lang.String)